Front-end/src/Home.jsx

```diff
@@ -235,6 +243,65 @@ const Home = () => {
                   options={options}
                   required
                 />
+                <label>Age</label>
+                <FormInput name="Age" placeholder="Age" min="18" required />
+                <label>State</label>
+                <select name="state" required>
+                  <option value="" selected="selected">
+                    Select a State
+                  </option>
+                  <option value="AL">Alabama</option>
+                  <option value="AK">Alaska</option>
+                  <option value="AZ">Arizona</option>
+                  <option value="AR">Arkansas</option>
+                  <option value="CA">California</option>
+                  <option value="CO">Colorado</option>
```

Comment:
Consider saving these state name to an map, and iterate the map with the element. You could refactor this part after MVP done.

Reply:
Sounds great